source/Assets/Plugins/Didomi/Scripts/Android/AndroidDidomi.cs

```diff
@@ -616,5 +634,12 @@ private static void MapNullValuesToJava(object[] args)
                 }
             }
         }
+
+        /**
+         * Calls 'setupUI' (Must be call from UI thread)
```

Comment:
"must be called" instead of "call"

Reply:
in addition to Philémon's suggestion, another minor suggestion: `Calls 'setupUI' native method from current thread (must be called from UI thread)`